source/fab/language/__init__.py

```diff
@@ -9,8 +9,8 @@
 from pathlib import Path
 from typing import List
 
-from fab.database import StateDatabase
-from fab.reader import TextReader
+from fab.database import StateDatabase, FileInfoDatabase
+from fab.reader import TextReader, TextReaderAdler32
```

Comment:
I'm not sure I like having `FileInfoDatabase` in `fab.database`. What do you think about a new module which contains both it and `TextReaderAdler32`? That way the two components which work together live together and neither sit in their parents modules.

Reply:
I see where you're coming from - as a separate ticket?

Reply:
I've left this alone for now - in part because we've actually now _lost_ `TextReaderAdler32` owing to one of the comments further down.  We could move `FileInfoDatabase` into its own module but I'm a little unsure